test/conformance/chainsaw/cli/two-results-with-namespace/kyverno-test.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: cli.kyverno.io/v1alpha1
+kind: Test
+metadata:
+  name: kyverno-test.yaml
+policies:
+- policy.yaml
+resources:
+- resources.yaml
+results:
+- kind: Pod
+  policy: default/test-jmespath
+  resources:
+  - test-invalid
+  result: fail
+  rule: test-jmespath
+- kind: Pod
+  policy: test-jmespath
```

Comment:
I would expect the presence of this to throw another error. Are multiple errors returned if multiple are found or only the first? The fact that the value of this field is `test-jmespath` means the CLI is looking for a `ClusterPolicy`. There is no such ClusterPolicy here.